src/features/usage-consolidation-accordion/summary-table/column-properties.js

```diff
@@ -21,11 +27,20 @@ const DEFAULT_SUMMARY_TABLE_COLUMN_MAPPING = {
   [DEFAULT_SUMMARY_TABLE_COLUMNS.UC_ACTIONS]: null,
 };
 
-export const getSummaryTableColumnProperties = (intl, customProps = {}) => {
+const DEFAULT_FORMATTER = (currency) => ({
+  cost: rowData => formatValue(rowData.cost, (value) => formatCost(currency, value)),
+  costPerUse: rowData => formatValue(rowData.costPerUse, (value) => formatCost(currency, value)),
+  usage: rowData => formatValue(rowData.usage, (value) => <FormattedNumber value={value} />),
+});
+
+export const getCostPerUseFormatter = (currency) => DEFAULT_FORMATTER(currency);
```

Comment:
I think that is not the best way, It's better to omit this function(in my opinion) or just declare it for `getCostPerUseFormatter`
```suggestion
export const getCostPerUseFormatter = DEFAULT_FORMATTER;
```